modules/userId/index.js

```diff
@@ -48,6 +48,20 @@
  * @type {string}
  */
 
+/**
+ * @property
+ * @summary use a predefined domain override for cookies or provide your own
+ * @name Submodule#domainOverride
```

Comment:
i don't see this property used anywhere...do we still need it?

Reply:
I discovered it when doing more research. It was already in use in `setStoredValue` and in pubCommonIdSubmodule so I just added the documentation for it. So yes, it seems confusing because I just added the doc line about existing code that I did not modify

Reply:
ah ok. it looked familiar but i thought maybe it was from a previous version of this PR. all good, thanks for adding the doc!